charts/internal/gvisor/values.yaml

```
@@ -0,0 +1,9 @@
+images:
+  runtime-gvisor: "image-repository:image-tag"
+
+config:
+  binFolder: /var/bin/containerruntime
```

```suggestion
  binFolder: /var/bin/containerruntimes
```

Just to match the example in g/g, but it'll be overridden from outside anyways.